src/index.html

```diff
@@ -11,6 +11,19 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form" data-qa="big">
+      <input type="text"
+        class="search-form__input"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+      >
```

Comment:
```suggestion
      <input 
        type="text"
        class="search-form__input"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
      >
```
Consider fixing indents according to the code style guide